distributed/protocol/serialize.py

```diff
@@ -568,6 +568,11 @@ def _deserialize_bytes(header, frames):
     return b"".join(frames)
 
 
+@dask_deserialize.register(memoryview)
+def _serialize_memoryview(header, frames):
+    return memoryview(b"".join(frames))
```

Comment:
Is it worth special casing length-1 `frames`, since then you can just return `frames[0]` and avoid the copy? Maybe?

Reply:
Good point

Reply:
Also true for `_deserialize_bytes`, right?